Batch.roboFontExt/lib/webFormats/__init__.py

```diff
@@ -179,7 +179,7 @@ class TTHAutoHintSettings(BaseWindowController):
 
     def __init__(self, parentWindow):
 
-        data = getExtensionDefault(self.identifier, dict())
+        data = getExtensionDefault(settingsIdentifier, dict())
```

Comment:
why changing this identifier? this is for the autohint settings and is different from other batch settings

Reply:
class variables are accessible with self.<name>

see https://github.com/typemytype/batchRoboFontExtension/blob/master/Batch.roboFontExt/lib/webFormats/__init__.py#L178

Reply:
Ah, right. Any hunch as to why the report was inaccurate before this change?
https://github.com/typemytype/batchRoboFontExtension/issues/60

Reply:
How's fafe703ac4afe0431282394da69f44cdecffe414? 
The identifier for specifically autohint settings seemed to want to be used outside the TTFAutoHint settings window class, so I made it a global variable and applied it everywhere. The report is accurate now; does it seem right in the code to you?